Dev

harbieism opened the pull request:
Revised parts of the algorithms to get data from user-defined shapes and added tests for them.

harbieism committed:
Added a test for get_field_for_area intersections

harbieism committed:
Added a contains test for get_field_for_area

harbieism committed:
Changed highest_resolution_for_data

highest_resolution_for_data will now work even if
the centroid of the area_to_get is not inside a
boundary that contains field_to_get.  It will
only return None if there are no boundaries that
contain field_to_get in boundary_list.

harbieism committed:
Merge branch 'master' of github.com:harbieism/HealthAround.me into get_field_for_area_tests

harbieism committed:
Added some trash files to .gitignore.

harbieism committed:
Merge branch 'master' of github.com:harbieism/HealthAround.me into get_field_for_area_tests

harbieism committed:
Added a test for get_field_for_area

Added a test for get_field_for_area for when area
includes boundaries that are not do not have
field_to_get.

harbieism committed:
Merge branch 'test/get_field_for_area/outside' into get_field_for_area_tests

harbieism committed:
Added test for get_field_for_area for lack of data

harbieism committed:
Merge branch 'test/getFieldForArea/noData' into get_field_for_area_tests

harbieism committed:
Merge branch 'get_field_for_area_tests' into adding_tests

harbieism committed:
Setup for writing highest_resolution_for_data test

harbieism committed:
Test setup retooled to make it more modular

harbieism committed:
Added tests for when area isn't in any boundary

harbieism committed:
Changed formatting, added tests for states.

harbieism committed:
Added tests for when data is present at all levels

harbieism committed:
Added tests for when no data is available.

harbieism committed:
Setup tests for get_field_for_area_percent.

harbieism committed:
Removed unused fields from the census test models

harbieism committed:
Fixed a bug with the reduced census fields

harbieism committed:
Fixed a couple of bugs, added first AreaPer test.

harbieism committed:
Added more tests for field_per.

harbieism committed:
Failure case of get_field_for_area returns None

jwhitlock commented:
Code is merged and live on http://healtharound.me  Do we need to re-import data?

harbieism commented:
No.  I just changed some of the utilities and added tests for them.

On Wed, Jul 23, 2014 at 11:48 AM, John Whitlock notifications@github.com
wrote:

> Code is merged and live on http://healtharound.me Do we need to re-import
> data?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CivicNinjas/HealthAround.me/pull/32#issuecomment-49901421
> .
